dist/post_run/index.js

```diff
@@ -6811,14 +6811,8 @@ function runLint(lintPath, patchPath) {
         }
         const userArgs = core.getInput(`args`);
         const addedArgs = [];
-        const userArgNames = new Set();
-        userArgs
-            .split(/\s/)
-            .map((arg) => arg.split(`=`)[0])
-            .filter((arg) => arg.startsWith(`-`))
-            .forEach((arg) => {
-            userArgNames.add(arg.replace(`-`, ``));
```

Comment:
I think that the real problem is here: the `replace` must be replaced by a trim prefix.